JASP-Desktop/po/jasp_nl.po

```diff
@@ -16078,10 +15845,2124 @@ msgctxt "RibbonModel|"
+msgstr "Normale a Priori Verdeling voor de Log Odds Ratio"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "μ:"
+msgstr "μ:"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "σ:"
+msgstr "σ:"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Descriptives"
+msgstr "Beschrijvende statistiek"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Plots"
+msgstr "Grafieken"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Prior and posterior"
+msgstr "Prior en posterior"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Sequential analysis"
+msgstr "Sequentiële analyse"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Prior"
+msgstr "Prior"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Bayes factor robustness check"
+msgstr "Bayes factor robuustheidsevaluatie"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Order"
+msgstr "Volgorde"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Compare to best model"
+msgstr "Vergelijk met het beste model"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Compare to null model"
+msgstr "Vergelijk met het null model"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Advanced Options"
+msgstr "Geavanceerde Opties"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Prior Model Probability"
+msgstr "Prior Model Kans"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Log odds ratio = 0"
+msgstr "Log odds ratio = 0"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Log odds ratio > 0"
+msgstr "Log odds ratio > 0"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Log odds ratio < 0"
+msgstr "Log odds ratio < 0"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Log odds ratio ≠ 0"
+msgstr "Log odds ratio ≠ 0"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Sampling"
+msgstr "Steekproef aan het nemen"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "No. samples"
+msgstr "Aantal steekproeven"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Robustness Plot"
+msgstr "Robuustheid Diagram"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "No. Steps"
+msgstr "Aantal Stappen"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "Step Range"
+msgstr "Stappenbereik"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "lower:"
+msgstr "onder:"
+
+msgctxt "SummaryStatsABTestBayesian|"
+msgid "upper:"
+msgstr "boven:"
+
+#, qt-format
+msgctxt "AnalysisForm|"
+msgid ""
+"Component %1 was loaded with the wrong type of value and has been reset to "
+"its default value."
+msgstr "Component %1 heeft een verkeerde type, de standaardwaarde is herstelt."
+
+msgctxt "AnalysisForm|"
+msgid ""
+"Many components were loaded with the wrong type of value and have been reset "
+"to their default values."
+msgstr "Verschillende componenten hebben een verkeerde type, de standaardwaarden zijn herstelt."
+
+msgctxt "ABTestBayesian|"
+msgid "Prior Model Probability"
+msgstr "Prior Model Kans"
+
+msgctxt "auditBayesianWorkflow|"
+msgid ""
+msgstr ""
+
+msgctxt "auditClassicalWorkflow|"
+msgid ""
+msgstr ""
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid ""
```

Comment:
@FBartos  you really don't need to put `qsTr("")` how are people going to translate `""` ?